### DISCUSSION TITLE
Dịch grid search, random search

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#27] [Terminology] [answered]
Mình đề xuất dịch 2 từ này là:
* `grid search` -> `tìm kiếm dạng lưới`
* `random search` -> `tìm kiếm ngẫu nhiên`

Mọi người thấy ổn không nhỉ? @mlbvn/handson-ml2

**Comment by cuongvng** [comment]
`grid search`: `tìm kiếm lưới`?

**Reply by minhduc0711** [reply]
cũng được, nhưng mình nghe từ này thấy hơi thiếu thiếu

**Reply by cuongvng** [reply]
em lại nghe `dạng lưới` hơi thừa 😄

**Comment by quangnhat185** [comment]
Mặc dù rất muốn dịch là `thiên la địa võng` hoặc `lưới trời lòng lọng` , mình sẽ vote cho `tìm kiếm dạng lưới`.

Cơ mà `Sử dụng thiên la địa võng để tìm siêu tham số tối ưu` nghe cool phết.

**Comment by minhduc0711** [comment, marked as answer]
chốt nhé mọi người
* `grid search` -> `tìm kiếm dạng lưới`
* `random search` -> `tìm kiếm ngẫu nhiên`
